README.md

```diff
@@ -81,7 +93,7 @@ body that says "hello".
 
 Your Answer:
 ```js
-// your code here
+$("#greeting").on("click", this.append("<p>hello</p>"));
```

Comment:
This would work perfectly fine! I think the question was asking you to append it to the body, so the solution looks like this: 

```
$("#greeting").on("click", function(){
  $("body").append("<p>hello</p>");
});
```